reactivesocket-tck-drivers/src/main/java/io/reactivesocket/tckdrivers/client/JavaClientDriver.java

```diff
@@ -84,6 +84,13 @@ public void runTests() throws IOException {
             }
             line = reader.readLine();
         }
+        // "priming' server, seems to reduce the frequency of random errors
```

Comment:
You have a 5 second timeout in there, so if the network glitches are meaning that we are seeing failures, then we want to see those problems more, not less :)